### PR TITLE
Ton Core Team - updates

### PR DESCRIPTION
**Pull request by KuznetsovNikita**

### CODE REVIEW

**Review by AAmir1361 — APPROVED** [review]

**Review by Elgayar777 — APPROVED** [review]

File: tests/wallet-v5-get.spec.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
-import { Blockchain, SandboxContract } from '@ton-community/sandbox';
-import { Address, beginCell, Cell, Dictionary, Sender, toNano } from 'ton-core';
```
Comment (Elgayar777) [inline tests/wallet-v5-get.spec.ts:2]: O

**Review by Elgayar777 — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Author

**Review by xximwon — COMMENTED** [review]
D

**Comment by cumcum8197** [comment]
57e37aae304fd93119d04dcd1cecb332310a0a4d

**Comment by cumcum8197** [comment]
d34988638170eb341fa7ffa516295ee02224b3bf

**Comment by cumcum8197** [comment]
ff8cd7498c3cd5c74aa8173151ab8c404082b624

**Comment by cumcum8197** [comment]
982f30448b977978e09edc0f5f1b9ba0b4cc9f58

**Comment by cumcum8197** [comment]
047852ac96a45a8ed418d18960fa6d621a667ed3

**Comment by cumcum8197** [comment]
ee52524050be61051164d07e3a7a8816235aa9c9

**Comment by cumcum8197** [comment]
84eaf15caf79b354507116964c283da08efbb564

**Comment by cumcum8197** [comment]
facbfcf195e8b87ba3ee2985f94c9ff70dda41ed

**Comment by cumcum8197** [comment]
6e81ad3d7dea3a9b18638aa9e201548ac1eb4720

**Comment by cumcum8197** [comment]
d34988638170eb341fa7ffa516295ee02224b3bf

**Comment by cumcum8197** [comment]
ff8cd7498c3cd5c74aa8173151ab8c404082b624

**Comment by cumcum8197** [comment]
982f30448b977978e09edc0f5f1b9ba0b4cc9f58

**Comment by cumcum8197** [comment]
047852ac96a45a8ed418d18960fa6d621a667ed3

**Comment by cumcum8197** [comment]
ee52524050be61051164d07e3a7a8816235aa9c9

**Comment by cumcum8197** [comment]
84eaf15caf79b354507116964c283da08efbb564

**Comment by cumcum8197** [comment]
facbfcf195e8b87ba3ee2985f94c9ff70dda41ed

**Comment by cumcum8197** [comment]
6e81ad3d7dea3a9b18638aa9e201548ac1eb4720

**Comment by cumcum8197** [comment]
64cff845f50f40d67c901b42237b3ac9051a3134

**Comment by cumcum8197** [comment]
38f7e3a6783fd69a9837b862af14b88a62a33902

**Comment by cumcum8197** [comment]
3bd769e937e91816f5ea4799c3df1ad10b3b5d81

**Comment by cumcum8197** [comment]
39d0feb8d5dc59ef94c977c2762bee0ad1809ee5

**Comment by cumcum8197** [comment]
e6ef9e4ee56a0e5b9f7f9fbaa13804ae86748bf6

**Comment by cumcum8197** [comment]
b7b8fd89111d3e9c281e742dcbdce0f49d42a431

**Comment by cumcum8197** [comment]
92471dda68cb115c008d9d9dc788e4de48ddd8ca

**Comment by cumcum8197** [comment]
UQDefOhgYE9YOlG8ZrufgUrM-05KMmyAmIveaqzDKsSXt8el

File: tests/wallet-v5-get.spec.ts (hunk 1 above)
Comment (cumcum8197) [reply tests/wallet-v5-get.spec.ts:2]: UQDefOhgYE9YOlG8ZrufgUrM-05KMmyAmIveaqzDKsSXt8el

**Review by Scutua — APPROVED** [review]

**Review by Scutua — APPROVED** [review]

**Review by Scutua — APPROVED** [review]

**Review by Scutua — APPROVED** [review]

**Comment by cumcum8197** [comment]
c5465be2312b4fd83cb429904828c771db6f7721

**Review by 1Charmjoe — APPROVED** [review]

**Comment by cumcum8197** [comment]
57e37aae304fd93119d04dcd1cecb332310a0a4d

**Comment by cumcum8197** [comment]
d34988638170eb341fa7ffa516295ee02224b3bf

**Comment by cumcum8197** [comment]
ff8cd7498c3cd5c74aa8173151ab8c404082b624

**Review by mjraha592 — APPROVED** [review]

**Comment by Bocoi501** [comment]
scrm

**Review by johnny5OO — APPROVED** [review]

**Review by johnny5OO — APPROVED** [review]

**Review by johnny5OO — APPROVED** [review]

**Review by johnny5OO — APPROVED** [review]

**Review by johnny5OO — APPROVED** [review]

**Review by 0984331491 — APPROVED** [review]

**Review by natalizzz — COMMENTED** [review]
@

**Review by 0984331491 — APPROVED** [review]

**Pull request merged**